joomla/joomla-cms
# Update Jooa11y with latest Sa11y build

### Summary of Changes
This PR adds [Sa11y](https://github.com/ryersondmp/sa11y) as a dependency, ensuring seamless and automatic updates for future Joomla releases. 

- Sa11y is added as a dependency via `build/build-modules-js/settings.json`. 
- Addition of several new languages. 
     - New languages would be added by modifying `build/build-modules-js/settings.json`. 
     - Automatic language detection based on page language.
     - Note: Only the admin settings will utilize Joomla's PLG language system.
- New configuration/properties in admin settings to customize experience for content editors.

Related issue: https://github.com/joomla-projects/joomla-a11y-checker/issues/75

### Testing Instructions
- Ensure you have a fresh JS build via `npm install` 
- Navigate to "Accessibility Check" as you would normally. Accessibility Checker should appear in same spot.

### Actual result BEFORE applying this Pull Request
https://joomla-projects.github.io/joomla-a11y-checker/pages/errors.html

### Expected result AFTER applying this Pull Request

#### Editor preview
![Screenshot of Joomla Accessibility Checker using Accessibility Check preview on a German page.](https://github.com/joomla/joomla-cms/assets/26335725/0ea62dcd-38b4-4e54-9a6e-74e41b87ddd5)

#### Plugin settings
![New Joomla Accessibility Checker plugin admin settings.](https://github.com/joomla/joomla-cms/assets/26335725/0a09ae18-0367-45fc-b28c-5f8abe08592b)


### Link to documentations
Please select:
- [x] Pull Request link for manual.joomla.org: ([https://manual.joomla.org/docs/accessibility/testing](https://manual.joomla.org/docs/accessibility/testing))

**Is this page ([https://manual.joomla.org/docs/accessibility/testing](https://manual.joomla.org/docs/accessibility/testing)) something I can help update?**
